src/test/java/com/sap/sgs/phosphor/fosstars/data/github/UsesSignedCommitTest.java

```diff
@@ -13,95 +14,106 @@
 import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProject;
 import com.sap.sgs.phosphor.fosstars.tool.github.GitHubProjectValueCache;
 import java.io.IOException;
-import java.io.InputStream;
-import org.apache.http.HttpEntity;
-import org.apache.http.client.methods.CloseableHttpResponse;
-import org.apache.http.impl.client.CloseableHttpClient;
+import java.util.ArrayList;
+import java.util.List;
 import org.junit.Test;
+import org.kohsuke.github.GHCommit;
+import org.kohsuke.github.GHCommit.ShortInfo;
+import org.kohsuke.github.GHVerification;
 import org.kohsuke.github.GitHub;
 
 public class UsesSignedCommitTest {
 
   @Test
-  public void testUsesSignedCommitsTrue() throws IOException {
+  public void testCommitWithSignature() throws IOException {
     GitHub github = mock(GitHub.class);
-    GitHubProject project = new GitHubProject("spring-projects", "spring-integration");
-    UsesSignedCommits provider = new UsesSignedCommits(github, null);
+
+    UsesSignedCommits provider = new UsesSignedCommits(github);
     provider = spy(provider);
     when(provider.cache()).thenReturn(new GitHubProjectValueCache());
 
-    CloseableHttpClient httpClient = mock(CloseableHttpClient.class);
-    when(provider.httpClient()).thenReturn(httpClient);
+    GHCommit commit = mock(GHCommit.class);
+    List<GHCommit> list = new ArrayList<GHCommit>();
+    list.add(commit);
 
-    CloseableHttpResponse response = mock(CloseableHttpResponse.class);
-    when(httpClient.execute(any())).thenReturn(response);
+    ShortInfo info = mock(ShortInfo.class);
+    when(commit.getCommitShortInfo()).thenReturn(info);
 
-    HttpEntity entity = mock(HttpEntity.class);
-    when(response.getEntity()).thenReturn(entity);
+    GHVerification verification = mock(GHVerification.class);
+    when(info.getVerification()).thenReturn(verification);
+    when(verification.isVerified()).thenReturn(true);
 
-    try (InputStream content = getClass().getResourceAsStream("UsesSignedCommitTrue.json")) {
```

Comment:
Is this file still necessary? Please also check other ones which used to be used by the test.

Reply:
Yup deleted.